### DISCUSSION TITLE
Pull requests after a repository is made read-only?

### DISCUSSION THREAD

**Discussion by DavidatPlanet** [community/community#57572] [Repositories]
### Select Topic Area

General

### Body

As part of an upcoming effort, will be making a project repo read-only / archive. 

Question to GitHub / GitHub community: After the project is made read-only, will any open pull requests will be retained, or will the open-pull requests simply be canceled / dropped.

Thanks,
David Hinz
Sr. Diretor, Corporate Engineering
Planet

**Comment by danielalyoshin** [comment]
As stated in the [official GitHub documentation](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories):

> When a repository is archived, its issues, pull requests, code, labels, milestones, projects, wiki, releases, commits, tags, branches, reactions, code scanning alerts, comments and permissions become read-only.

Hope this helps!
